rockcraft/extensions/gunicorn.py

```diff
@@ -66,13 +66,20 @@ def gen_install_app_part(self) -> Dict[str, Any]:
     def _gen_parts(self) -> dict:
         """Generate the parts associated with this extension."""
         data_dir = get_extensions_data_dir()
+        stage_packages = ["python3-venv"]
```

Comment:
you can still use chisel with a non bare base, there is file dedup in any case, so not as important, but there is more _scripting_ with chisel

Reply:
That's true, but doing so will change python version for `python3.10` in `ubuntu@22.04` base to `python3.11`, which might be a breaking change in the extension.
I'd rather keep the default python version of jammy in the `ubuntu@22.04` base

Reply:
why do you say it's a breaking change?

Reply:
Going to `python3.11` is a breaking change to a degree because Python usually does some minor backwards incompatible things even within minor versions. They usually don't impact many people (I haven't checked the specifics for 3.10 -> 3.11).

The more convincing reason not to go to `python3.11` for Ubuntu 22.04 is that people would expect Python 3.10 if they are building on top of Ubuntu 22.04 since that is what ships with it

Reply:
And it also looks like for some reason we would be shipping `3.11.0rc1` which doesn't seem like what we want to do when `3.11.9` is out

Reply:
So few more elements:
Ubuntu Main python package is 3.10
The sliced package in python3.11 which comes from universe

I think for base `ubuntu@22.04` we should either use the apt package or the python3.10 sliced (which doesn't exists yet).

Reply:
We decided to slice python3.10 in Ubuntu 22.04 and use it. I close the thread.